src/_actions/subcategory.actions.js

```diff
@@ -26,6 +32,33 @@ export const loadSubCategories = () => {
  *
  * @return {object} type and payload
  */
-export const loadSubCategoriesSuccess = (subcategories) => {
-  return { type: LOAD_SUBCATEGORIES_SUCCESS, payload: subcategories };
-}
+export const loadSubCategoriesSuccess = subcategories => ({
```

Comment:
I forgot to ask this yesterday but are all action creators being used elsewhere? All of them are exported but only a few are used in the components. Please ensure that only action creators imported in a different file are exported. This comment applies to all action files.

For example, `createSubCategoryFailure ` should not be exported as it is used within the same file. You'll have to search the repo using `grep` or your editor search - whichever works best for you

Reply:
Hi @sachungo, the reason I am exporting the action creators is that they are being used in the test files. For instance, `createSubCategoryFailure` is being used in the `subcategory.reducer.test.js` file. Does this clear up the issue?

Reply:
@Arthur236 You don't necessarily have to export those functions to test them, you can always test that they are dispatched when the `create` method is executed. Something like:
```
expect.objectContaining({
   type:  LOAD_SUBCATEGORIES_SUCCESS
})
```

However, it seems this will complicate your life a bit so ignore for now

Reply:
Okay. I'll look into refactoring this in future